### PR TITLE
Add ReleaseTag methods

### PR DESCRIPTION
**Pull request by thgreasi**
Resolves: #491
Change-Type: minor
HQ: https://github.com/resin-io/hq/pull/281
HQ: https://github.com/resin-io/hq/issues/1213
Depends-On: https://github.com/resin-io/resin-api/pull/952

### CODE REVIEW

**Comment by thgreasi** [comment]
Mostly identical to device-tags but w/o the ability to fetch tags based on a unique release property.

File: lib/models/release.coffee [hunk 1]
```diff
@@ -180,6 +188,162 @@ getReleaseModel = (deps, opts) ->
...
+			# only doing this so that the user get a ResinReleaseNotFound
+			# error when the release doean't actually exist
+			exports.get(id, $select: 'id').get('id')
+			# start the request on parallel, since we don't need
+			# any extra data to wait for
+			exports.tags.getAll(
+				mergePineOptions
+					$filter: release: id
+				, options
+			)
```
Comment (pimterry) [inline lib/models/release.coffee]: We could do these two together, couldn't we? Can't you get the release by id, and expand to get the tags too? I think that will throw properly if the release doesn't exist, and let you grab the tags from there very easily as well.

File: tests/integration/models/release.spec.coffee [hunk 2]
```diff
@@ -115,3 +122,36 @@ describe 'Release Model', ->
 							build_log: 'web log'
 						]
 
+		describe.only 'resin.models.release.tags', ->
```
Comment (pimterry) [inline tests/integration/models/release.spec.coffee]: Don't commit this `.only`! This disables all of the rest of the test suite

Comment (thgreasi) [reply tests/integration/models/release.spec.coffee]: Oh my...

**Pull request merged**